deployment/base/rbac/master-clusterrole.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: nfd-master
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - nodes
+  verbs:
+  - get
+  - patch
+  - update
+  # List only needed for --prune
+  - list
```

Comment:
patch it via `deployment/overlays/prune`?

Reply:
Changed accordingly: prune overlay now patches the clusterrole